ember-headless-table/src/plugins/column-reordering/plugin.ts

```diff
@@ -160,6 +160,58 @@ export class TableMeta {
     return this.columnOrder.orderedColumns;
   }
 
+  previousColumn = (referenceColumn: Column) => {
+    let visible = this.columns;
+    let referenceIndex = visible.indexOf(referenceColumn);
+
+    assert(
+      `index of reference column must be >= 0. column likely not a part of the table`,
+      referenceIndex >= 0
+    );
+
+    /**
+     * There can be nothing before the first column
+     */
+    if (referenceIndex === 0) {
+      return null;
+    }
+
+    return visible[referenceIndex - 1];
+  };
+
+  nextColumn = (referenceColumn: Column) => {
+    let visible = this.columns;
+    let referenceIndex = visible.indexOf(referenceColumn);
+
+    assert(
+      `index of reference column must be >= 0. column likely not a part of the table`,
+      referenceIndex >= 0
+    );
+
+    /**
+     * There can be nothing after the last column
+     */
+    if (referenceIndex > visible.length - 1) {
```

Comment:
Doesn't this need to be: 

```suggestion
    if (referenceIndex >= visible.length - 1) {
```

Reply:
if there are 5 columns
and reference index is 4 (max), there is no next column -- due to `4 > 5 - 1` being falsey... we hit line 198, and _accidentally_ get a falsey value, which is correct anyway

hm. this is copy and paste from the other implementation, but I'm going to fix this over in: https://github.com/CrowdStrike/ember-headless-table/pull/40
(and then delete this and the original implementation).

I do not like accidentally working. thanks for catching this!